cmd/bridge/main.go

```diff
@@ -551,6 +552,25 @@ func main() {
 		},
 	}
 
+	srv.KnativeEventSourceCRDLister = &server.ResourceLister{
+		BearerToken: resourceListerToken,
+		RequestURL: &url.URL{
+			Scheme: k8sEndpoint.Scheme,
+			Host:   k8sEndpoint.Host,
+			Path:   "/apis/apiextensions.k8s.io/v1beta1/customresourcedefinitions",
+			RawQuery: url.Values{
+				"labelSelector": {"duck.knative.dev/source=true"},
```

Comment:
`duck.knative.dev`  :)

Reply:
Yes we are fetching duck typed knative event sources